scripts/deploy.ts

```diff
@@ -13,42 +13,42 @@ import { DeployerFn } from '@ubeswap/hardhat-celo';
  *  (Yes, this is totally lame.)
  */
 
-// export const main: DeployerFn<any> = async() => {}  // uncomment for first-time contract compilation
+export const main: DeployerFn<any> = async() => {}  // uncomment for first-time contract compilation
```

Comment:
Undo this before merging? I have a PR coming up that will remove this file, though, so maybe doesn't matter.

Reply:
gah sorry